controllers/admin.js

```diff
@@ -270,7 +328,7 @@ exports.adminApiKeysPage = function (aReq, aRes, aNext) {
   // strategyListQuery
   tasks.push(function (aCallback) {
     Strategy.find({}, function (aErr, aStrats) {
-      var stored = nil();
+      var stored = nil();  // NOTE: Inconsistent usage of object creation here
```

Comment:
@sizzlemctwizzle Cc: @Zren 

Again... Is `nil` helper really needed here? So far I haven't seen any Object prototype lines in the controllers... but I haven't looked everywhere.